src/utils/Ponyfills.js

```diff
@@ -0,0 +1,15 @@
+// This file is intented to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
+export function convertUint8ArrayToString(array) {
+    if (window.TextDecoder !== undefined) {
+        return new TextDecoder().decode(array);
+    }
+
+    // Avoid the String.fromCharCode.apply(null, array) shortcut, which
+    // throws a "maximum call stack size exceeded" error for large arrays.
+    var s = '';
+    for (var i = 0, il = array.length; i < il; i++) {
+        s += String.fromCharCode(array[i]);
```

Comment:
This is not equivalent to `new TextDecoder().decode(…)` if there are values outside the 0–127 range; because `TextDecoder` defaults to UTF-8 when no encoding is given.

I don't know the usecases, and which inputs will be passed to this function, but's probably worth at least adding a comment (jsdoc?) that values are expected to all fall into the 0–127 range (i.e. be US-ASCII).

Maybe the fast path could also use `new TextEncoded('us-ascii')`? (it'll actually use `windows-1252`, but at least shows the intent)